packer/ubuntu-14.04_amd64-droplet.json

```diff
@@ -5,9 +5,9 @@
     "digitalocean_size": "{{env `DIGITALOCEAN_SIZE`}}",
     "digitalocean_image": "{{env `DIGITALOCEAN_IMAGE`}}",
     "mesos_version": "0.22.1-1.0.ubuntu1404",
-    "marathon_version": "0.8.1-1.0.171.ubuntu1404",
-    "consul_version": "0.5.1",
-    "weave_version": "v0.11.2",
+    "marathon_version": "v0.8.2",
```

Comment:
changing this version to the docker container version is gonna break the package install